fitbenchmarking/core/results_output.py

```diff
@@ -36,7 +37,16 @@ def save_results(options, results, group_name,
     :return: Path to directory of group results
     :rtype: str
     """
-    _, group_dir, supp_dir, fig_dir = create_directories(options, group_name)
+    _, group_dir, supp_dir, fig_dir, local_css_dir = \
```

Comment:
Seems that the first argument is not used anywhere, I suggest removing it if this is the case

Reply:
It looks like you're right.  I was wary of taking it out because I didn't know why it was there in the first place, but I don't see any reason for it (it's still built, but it's available as a member of `options`  anyway)